applications/MeshingApplication/custom_utilities/mmg_utility.h

```diff
@@ -2280,7 +2390,7 @@ class MmgUtility
     {
         double* step_data = pNode->SolutionStepData().Data(step);
         
-        // NOTE: This just works with tetrahedron (you are going to have poblems with anything else)
+        // NOTE: This just works with tetrahedron (you are going to have problems with anything else)
```

Comment:
why don't you make it generic? particularly since you would like to have prisms too